tests/integration/aof.tcl

```diff
@@ -216,10 +216,10 @@ tags {"aof"} {
         }
     }
 
-    ## Test that EXPIREAT is loaded correctly
+    ## Test that PEXPIREAT is loaded correctly
     create_aof {
         append_to_aof [formatCommand rpush list foo]
-        append_to_aof [formatCommand expireat list 1000]
+        append_to_aof [formatCommand pexpireat list 1000]
```

Comment:
can you explain why you did that change?

Reply:
With this change, `EXPIREAT` should never appear in AOF files. So I think we should adapt this test to test what's possible.